pkg/agent/cniserver/ipam/ipam_service.go

```diff
@@ -45,8 +45,13 @@ type IPAMConfig struct {
 	Ranges []RangeSet `json:"ranges,omitempty"`
 }
 
+type IPAMResult struct {
+	current.Result
+	VLANID *uint16
```

Comment:
I feel we can just use uint16 and use VLAN 0 to indicate no VLAN.

Reply:
Or define a type for VLANID, and a constant for NOVLAN.

Reply:
Refactored all vlanID from `*uint16` to `uint16`